### ISSUE TITLE
Metadata doesn't update on library refresh

### ISSUE THREAD

**Issue by techdragon** [gboudreau/XBMCnfoTVImporter.bundle#69]
When a library is refreshed metadata changes don't appear to be reflected in Plex.

**Comment by joserick** [comment]
Hello,
Apparently this problem is years old,
When you refresh the metadata in the library, they are uploaded to Plex specifically in the "com.plexapp.agents.xbmcnfotv" folder.

example:
/ Library / Application Support / Plex Media Server / Metadata / TV Shows / X / ALPHANUMERICNUMBER.bundle / Contents / com.plexapp.agents.xbmcnfotv /

But this data is not refreshed in the "_combined" and "_stored" folders.

Example:
/ Library / Application Support / Plex Media Server / Metadata / TV Shows / X / ALPHANUMERICNUMBER.bundle / Contents / _combined

which are the folders called Plex when displaying the data.

Could you check because the files in the "_stored" folder are not updated?

Thank you

PD: In my case I have changed the images of posters to prove, I have not done tests with the .nfo files (for the text) but I think the same should happen.

**Comment by SlrG** [comment]
Yes, it's a known problem. Sadly I don't know a solution, as the problem seems to be more on the plex' side of things. It refreshes the text but it keeps the images. The only way around this, is to move the movie out of the file structure, refresh, move it back and refresh it again. Then the cache gets cleaned up and the new images are shown. Alternatively the whole library can be deleted and recreated. If anybody knows a way to force plex to refresh its cached images from the agents side, please let me know.

**Comment by Squazz** [comment]
@SirG, @joserick, @techdragon 
Could this issue be closed as it seems a fix was merged 3 years ago? :) Or am I not understanding the merge or the issue?
